flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/api/FlinkClient.java

```diff
@@ -183,10 +183,10 @@ public void submitTopologyWithOpts(final String name, final String uploadedJarLo
 
 		/* set storm configuration */
 		if (this.conf != null) {
-			topology.getConfig().setGlobalJobParameters(new StormConfig(this.conf));
+			topology.getExecutionEnvironment().getConfig().setGlobalJobParameters(new StormConfig(this.conf));
 		}
 
-		final StreamGraph streamGraph = topology.getStreamGraph();
+		final StreamGraph streamGraph = topology.getExecutionEnvironment().getStreamGraph();
 		streamGraph.setJobName(name);
```

Comment:
Because FlinkTopology is not subclassed from StreamExecutionEnvironment anymore.